GrandCentralBoard/Widgets/Harvest/HarvestWidgetBuilder.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  HarvestWidgetBuilder.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-11.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import GrandCentralBoardCore
+
+
+final class HarvestWidgetBuilder: WidgetBuilding {
+    var name = "harvest"
```

Comment:
I haven't changed the widget builder name. It is somewhat generic, but in my opinion the change isn't required as long as we don't have any competing widgets.